pkg/scheduler/metrics/metrics.go

```diff
@@ -27,7 +27,7 @@ const (
 	// SchedulerSubsystem - subsystem name used by scheduler
 	SchedulerSubsystem = "scheduler"
 	// SchedulingLatencyName - scheduler latency metric name
-	SchedulingLatencyName = "scheduling_latencies_summary"
+	SchedulingLatencyName = "scheduling_latency_seconds"
 
 	// OperationLabel - operation label name
 	OperationLabel = "operation"
```

Comment:
I would prefer that we don't use `scheduling_algorithm` as a bucket name. "scheduling" adds no meaning in this context, as the subsystem name and metric name already contain it. "algorithm" is vague to anyone not familiar with our internal implementation details.

Maybe `select_node` instead?

Reply:
Renamed to "selecting_node"